.kiro/settings/mcp.json

```diff
@@ -0,0 +1,38 @@
+{
+    "mcpServers": {
+        "aws-infrastructure-as-code": {
+            "command": "uvx",
+            "args": [
+                "awslabs.aws-iac-mcp-server@latest"
+            ],
+            "env": {
+                "FASTMCP_LOG_LEVEL": "ERROR"
+            },
+            "disabled": false,
+            "autoApprove": [
+                "search_cdk_documentation",
+                "search_cdk_samples_and_constructs",
+                "search_cloudformation_documentation",
+                "cdk_best_practices",
+                "read_iac_documentation_page"
+            ]
+        },
+        "aws-mcp": {
+            "command": "uvx",
+            "timeout": 100000,
+            "transport": "stdio",
+            "args": [
+                "mcp-proxy-for-aws@latest",
+                "https://aws-mcp.us-east-1.api.aws/mcp",
+                "--metadata",
+                "AWS_REGION=us-east-1"
```

Comment:
Specifying `AWS_REGION` is not necessary as it's the default.

https://docs.aws.amazon.com/aws-mcp/latest/userguide/getting-started-aws-mcp-server.html#step-3-configure-mcp-client

Could you also add the source material (guides or docs) to the description of the PR?